hypertrace-trace-enricher/hypertrace-trace-enricher-impl/src/main/java/org/hypertrace/traceenricher/enrichment/enrichers/resolver/backend/JdbcBackendResolver.java

```diff
@@ -55,6 +55,11 @@ public JdbcBackendResolver(FQNResolver fqnResolver) {
         dbType = JDBC_EVENT_PREFIX;
       }
 
+      Optional<String> optionalDbType = DbAttributeUtils.getDbTypeForOtelFormat(event);
```

Comment:
Should we consider all sql dbs from Otel  as `BackendType.JDBC` ?

Reply:
That's the assumption I'm going ahead with, they definitely don't fall under redis, mongo, http, grpc, rabbitmq?

Reply:
But, wouldn't it requires to check any other specific JDBC related attribute?

Reply:
I don't think there are other decisive parameters.